mygame/app/species.rb

```diff
@@ -24,23 +24,596 @@
   source_y: 4
 ).freeze
 
+GREEN_BACK_SPRITE = EMOJI_BACK_BASE_SPRITE.merge(
+  source_x: 32,
+  source_y: 4
+).freeze
+
+PURPLE_BACK_SPRITE = EMOJI_BACK_BASE_SPRITE.merge(
+  source_x: 48,
+  source_y: 4
+).freeze
+
+BLUE_BACK_SPRITE = EMOJI_BACK_BASE_SPRITE.merge(
+  source_x: 80,
+  source_y: 4
+).freeze
+
 SPECIES = {
-  wink: {
-    name: 'WINK',
-    types: [:sassy, :sexy],
+  slightly_smiling: {
+    name: "Slightly Smiling",
```

Comment:
We will definitely need to decide some kind of max name length for those emojis... might otherwise be difficult to display them on the screen...

Reply:
totally agree. I just picked up the convention from actual emojis.